deploy/sre-build-test/50-source-build-test.CronJob.yaml

```
@@ -103,7 +103,7 @@ spec:
               echo "$(date): Waiting for build to complete."
               while :
               do
-                ST=$(oc -n "${NS}" get build -o custom-columns=STATUS:.status.phase --no-headers)
+                ST=$(oc -n "${NS}" get build -o custom-columns=STATUS:.status.phase --no-headers | tail -1)
```

```suggestion
                ST=$(oc -n "${NS}" get build -o custom-columns=STATUS:.status.phase --sort-by=.status.startTimestamp --no-headers | tail -1)
```

Great catch! Can we add in an explicit sort by the starting timestamp so that we are sure the last one listed is the most recent?